Inspicio/Inspicio.csproj

```diff
@@ -58,5 +72,6 @@
     <Folder Include="Migrations\" />
     <Folder Include="Migrations\" />
     <Folder Include="Migrations\" />
+    <Folder Include="Migrations\" />
```

Comment:
Not sure what's going on here, but the Migrations folders is mentioned multiple times in the project file - I'd expect it to only be there once.